docker-hub/builds/index.md

```diff
@@ -40,7 +44,7 @@ queued to be run later.
 
 > **Note**
 >
-> The total number of pending builds in the queue is limited to 30 and further requests will be discarded.
+> The total number of pending builds in the queue is limited to 30 and further requests will be discarded. The number of parallel builds has increased to 5 for Pro and 15 for Team.
```

Comment:
I think we should rather state this as the current state, because for most new users they won't know that it used to be lower. So we can drop "increased"